graphql/resolvers.go

```diff
@@ -357,6 +359,49 @@ func (r *queryResolver) TaskFiles(ctx context.Context, taskID string) ([]*Groupe
 	return groupedFilesList, nil
 }
 
+func (r *mutationResolver) SetPriority(ctx context.Context, taskID string, priority int) (*restModel.APITask, error) {
+	t, err := task.FindOneId(taskID)
+	if err != nil {
+		return nil, ResourceNotFound.Send(ctx, err.Error())
+	}
+	if t == nil {
+		return nil, errors.Errorf("unable to find task %s", taskID)
```

Comment:
all gql errors should have a code associated with them. this error would also be `ResourceNotFound`